src/taipy/core/data/_data_model.py

```diff
@@ -52,6 +53,7 @@ def from_dict(data: Dict[str, Any]):
             parent_ids=data["parent_ids"],
             last_edit_date=data.get("last_edit_date", data.get("last_edition_date")),
             job_ids=data["job_ids"],
```

Comment:
and here too?

Reply:
I think we must still keep it for the same reason.